crates/sui-protocol-config/src/lib.rs

```diff
@@ -3249,8 +3278,11 @@ impl ProtocolConfig {
                 75 => {
                     if chain != Chain::Mainnet {
                         cfg.feature_flags.passkey_auth = true;
+                        cfg.feature_flags.record_additional_state_digest_in_prologue = true;
```

Comment:
is there a reason to turn this on in testnet separately from whatever configuration we eventually add to turn on the whole execution time estimator system?

Reply:
no, that was a mistake. turning it off for testnet as well